assets/js/components/PaymentRequest/list.vue

```diff
@@ -1,10 +1,35 @@
 <template>
   <div class="list">
     <h2>Payment Requests</h2>
+    <div class="d-flex flex-row">
+      <request-row v-for="request in orderedRequests" :key="request.id" :request="request"></request-row>
+    </div>
   </div>
 </template>
 <script>
+import ApiMixin from "../../mixins/Api";
+import RequestRow from './request-row'
+import _ from 'lodash';
 export default {
-  name: 'payment-request-list',
-}
+  mixins: [ApiMixin],
+  name: "payment-request-list",
+  data: function(){
+    return {
+      requests : []
+    };
+  },
+  computed: {
+    orderedRequests() {
+      return _.orderBy(this.requests, (x)=>x.expense_instance.created_at)  
+    }
+  },
+  components:{
+    'request-row': RequestRow
+  },
+  mounted: function() {
+    this.sendRequest(`payment_requests/${this.$store.state.user.id}`, "GET").then(x=>{
```

Comment:
IMO, we should have `/payment_requests` just return the requests for the current logged-in user. Makes things a bit simpler + makes things more secure

That's what I did here on the expenses index page: https://github.com/UtilShare/util-share/blob/master/lib/utilshare_web/controllers/expense_controller.ex#L15